src/Bootstrap/less/theme/page-display-package-v2.less

```
@@ -324,4 +324,35 @@
       }
     }
   }
+
+  .body-tabs {
+
+    .nav-tabs > li.active > a {
+      border-bottom-color: #0078D4;
+      border-bottom-width: 2px;
+      font-weight: bold;
+    }
+
+    .nav-tabs > li > a {
+      border-left: 0px;
+      border-right: 0px;
+      border-top: 0px;
+      font-size: 14px;
+      font-family: @font-family-base;
+      color: #323130;
+    }
+  }
+
+  .body-tab-content {
+    padding-top: 30px;
+
+    h1:first-child,
+    h2:first-child,
+    h3:first-child,
+    h4:first-child,
+    h5:first-child,
+    h6:first-child {
+        margin-top:0px;
```

It looks like we use 2 spaces for indentation on this file:

```suggestion
      margin-top: 0px;
```

---

done 👍